tests/fast-integration/logging/helpers.js

```diff
@@ -1,10 +1,14 @@
 const {assert} = require('chai');
-
+const k8s = require('@kubernetes/client-node');
 const {sleep} = require('../utils');
 const {
   queryLoki,
+  lokiSecretData,
+  lokiPersistentVolumClaim,
+  getVirtualServices,
 } = require('./client');
 
+// checkLokiLogs used directly in Commerce Mock tests.
```

Comment:
I would like to extend this to have the logs checked from all namespaces (atlease kyma-system and kyma-integtation)